app/styles/base.css

```diff
@@ -247,6 +247,16 @@ select:active {
   font-weight: normal;
 }
 
+#noVNC_fallback_errormsg .noVNC_message {
+  margin-left: auto;
+  margin-right: auto;
+  display: inline-block;
```

Comment:
Easier to get centering without an explicit size set.  It occurs to me that the `margin-left` and `margin-right` are unnecessary here.